README.md

```diff
@@ -91,20 +91,24 @@ AndroidAttestationConfiguration(
             signatureDigests = listOf("NLl2LE1skNSEMZQMV73nMUJYsmQg7=".encodeToByteArray()),
             appVersion = 2, //with a different versioning scheme
             androidVersionOverride = 130000, //so we need to override this
-            patchLevelOverride = PatchLevel(2023, 6), //also override patch level
+            patchLevelOverride = PatchLevel(2023, 6, maxFuturePatchLevelMonths = 2), //also override patch level and
+                                                                    // consider patch levels from 2 months in the future
+                                                                    // as valid 
+                                                                    // default = 1
+                                                                    // null means any future patch level is ok     
             trustAnchorOverrides = setOf(extraTrustedRootPubKey) // set this app to require
                                                                  // a custom root for trust for the
                                                                  // attestation certificate chain
         )
     ),
     androidVersion = 110000,                //OPTIONAL, null by default
-    patchLevel = PatchLevel(2022, 12),      //OPTIONAL, null by default
+    patchLevel = PatchLevel(2022, 12),      //OPTIONAL, null by default, maxFuturePatchLevelMonths defaults to 1
```

Comment:
but where is `maxFuturePatchLevelMonths` in this configuration line?

Reply:
nowhere, wich is why it defaults to 1. I amended the comment